DictionaryTags_MNE.txt

```diff
@@ -175,11 +175,13 @@ mne_rt_command              3700 ? "realtime command"
 mne_rt_client_id            3701 ? "realtime client"
 
 #
-# 3710... MNE epochs bookkeeping
+# 3800... MNE epochs bookkeeping
```

Comment:
I suppose the range for MNE is 3500-3999 as is said at the top of this file. These values are used in ``mne.io.constants``, so they were just wrong here.